hw/ip/otbn/doc/_index.md

```diff
@@ -604,13 +623,41 @@ When it executes this instruction, OTBN:
   </tbody>
 </table>
 
-## Device Interface Functions (DIFs)
 
-{{< dif_listing "sw/device/lib/dif/dif_otbn.h" >}}
+# Writing OTBN applications {#writing-otbn-applications}
 
-## Register Table
+OTBN applications are (small) pieces of software written in OTBN assembly.
+The full instruction set is described in the [ISA manual]({{< relref "hw/ip/otbn/doc/isa" >}}), and example software is available in the `sw/otbn` directory of the OpenTitan source tree.
 
-{{< registers "hw/ip/otbn/data/otbn.hjson" >}}
+## Toolchain support
```

Comment:
This is good. I wonder whether we should be trimming down the README in hw/ip/otbn/util and just pointing here?

Reply:
Oh, I just realized we had not only the README, but also the otbn_sw.md user guide on "Writing and building software for OTBN". Since the OTBN docs are meant to be a reference-style guide, I kept the content in the toolchain section as short as it was, and linked to the user guide in the beginning of the section.

The README section "Build OTBN software" is already quite short and actionable (and links to the relevant documentation in the user guide), so we can probably leave it as-is?